nisystemlink/clients/auth/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._additional_models import AuthResponse
```

Comment:
Is the intention here to only re-export the models that the minimal implementation of the client actually uses? That makes sense to me.

Reply:
Yes, and consistent with Dataframe client models. On the other hand, exporting everything has a side effect of adding the JsonModel and Field classes to be included in the readthedocs automated generation.